tests/test_charm.py

```diff
@@ -64,14 +64,14 @@ def test_alerting_config_is_updated_by_alertmanager_relation(self, push):
         self.assertEqual(alerting_config(config), None)
         push.reset_mock()
 
-        # check alerting config is updated when a alertmanager joins
-        self.harness.update_relation_data(rel_id,
-                                          'alertmanager',
-                                          {
-                                              'addrs': '["192.168.0.1:9093"]'
-                                          })
-        config = push.call_args[0]
-        self.assertEqual(alerting_config(config), SAMPLE_ALERTING_CONFIG)
+        # # check alerting config is updated when a alertmanager joins
+        # self.harness.update_relation_data(rel_id,
```

Comment:
Same question about when not just delete rather than commenting it out. PS: I have not check test coverage since this is a draft. I trust your workmanship here.

Reply:
removed alertmanager test